mypy/subtypes.py

```diff
@@ -1261,14 +1261,87 @@ def find_member(
     is_operator: bool = False,
     class_obj: bool = False,
     is_lvalue: bool = False,
+) -> Type | None:
```

Comment:
Most of the remaining performance regression comes from `find_member`. Would it be feasible to add a fast path that could be used in the majority of (simple) cases? This only makes sense if the semantics would remain identical. We'd first try the fast path, and if it can't be used (not a simple case), we'd fall back to the general implementation that is added here (after `from mypy.checkmember import ...`). The fast path might look a bit like `find_member_simple`.

That fast path might cover access to normal attribute/method via instance when there are no self types or properties, for example. Maybe we can avoid creating `MemberContext` and using `filter_errors`.